op-node/p2p/node.go

```diff
@@ -6,6 +6,10 @@ import (
 	"fmt"
 	"strconv"
 
+	"github.com/ethereum-optimism/optimism/op-node/p2p/store"
+
```

Comment:
weird goimports formatting

Reply:
Pushed a commit to tidy these up so we can unblock further work.